src/swap.ts

```diff
@@ -1,10 +1,16 @@
+import { AxiosResponse } from "axios";
 import { BigNumber } from "bignumber.js";
 import { BitcoinWallet } from "./bitcoin_wallet";
-import { Cnd, LedgerAction, SwapDetails } from "./cnd";
+import { Cnd, LedgerAction, SwapDetails } from "./cnd/cnd";
+import { Field } from "./cnd/siren";
 import { EthereumWallet } from "./ethereum_wallet";
-import { Field } from "./siren";
-import { sleep, timeoutPromise, TryParams } from "./timeout_promise";
+import { sleep, timeoutPromise, TryParams } from "./util/timeout_promise";
 
+/**
+ * A state-ful class that represents a single swap.
```

Comment:
```suggestion
 * A statefull class that represents a single swap.
```

Reply:
I think neither of us are correct: it is `stateful`.